commands/upgrade/cluster/command_test.go

```diff
@@ -8,30 +8,31 @@ import (
 
 	"github.com/Jeffail/gabs"
 	"github.com/giantswarm/gscliauth/config"
-	"github.com/spf13/afero"
-
+	"github.com/giantswarm/gsctl/commands/errors"
 	"github.com/giantswarm/gsctl/testutils"
+	"github.com/google/go-cmp/cmp"
+	"github.com/spf13/afero"
```

Comment:
Can you check the imports in the PR? The `gsctl` imports should be the third block. See https://github.com/giantswarm/fmt/blob/master/go/imports.md.

Reply:
Thanks, done